Add celery, lz4 files (#139)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/celery.md

---
name: Celery
category: Messaging/Comms
description: Celery is a simple, flexible, and reliable distributed system to process vast amounts of messages, while providing operations with the tools required to maintain such a system.
download_url: https://github.com/celery/celery/releases
works_on_arm: true
supported_minimum_version:
    version_number: 4.0.0rc3
    release_date: 09/07/2016

optional_info:
    homepage_url: https://docs.celeryq.dev/en/stable/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content: https://community.arm.com/arm-research/b/articles/posts/scalable-hyperparameter-tuning-for-automl
        partner_content:  

>         official_docs: https://docs.celeryq.dev/en/v5.3.6/getting-started/index.html

Author: ```suggestion
        official_docs: https://docs.celeryq.dev/en/stable/
```

    arm_recommended_minimum_version:
        version_number: 
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum: 

>     other_info: Celery is not architecture-specific and none-any wheels are released on [PyPI](https://pypi.org/project/celery/#files).

Author: ```suggestion
    other_info: Celery is platform independent and wheels for same are released on [PyPI](https://pypi.org/project/celery/#files).
```
